app/src/components/fields/HorizontalSplitFormComponent.tsx

```diff
@@ -17,7 +17,7 @@ export interface IHorizontalSplitFormComponentProps {
    * @type {string}
    * @memberof IHorizontalSplitFormComponentProps
    */
-  summary?: string;
+  summary?: string | ReactNode;
```

Comment:
suggestion: ReactElement?
https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement